tool/sync_default_gems.rb

```diff
@@ -420,7 +419,10 @@ def sync_default_gems(gem)
       cp_r("#{upstream}/include/yarp/.", "yarp")
       cp_r("#{upstream}/include/yarp.h", "yarp")
 
-      rm("yarp/config.h")
+      cp_r("#{upstream}/config.yml", "tool/")
+      cp_r("#{upstream}/templates", "tool/")
```

Comment:
Seems copying YARP specific files to the common directory?
Really intentional?

Reply:
No, this should live under `tool/yarp`. I'll mark this as a draft until it's fixed.

Reply:
I changed my mind a little and put that under `yarp` instead.